src/Stripe.net/Services/StripeRequestOptions.cs

```diff
@@ -5,5 +5,8 @@ public class StripeRequestOptions
         public string ApiKey { get; set; }
         public string StripeConnectAccountId { get; set; }
         public string IdempotencyKey { get; set; }
+        // This is used specifically for Ephemeral Keys as they have to be created
+        // with a specific API version. Don't use it for anything else.
+        public string EphemeralKeyStripeVersion { get; set; }
```

Comment:
I don't think this ephemeral key specific field should be in this broadly used class. Can't you have each service method add in a normal Stripe version when they're called into?

Reply:
Or I guess there is no other `StripeVersion` option. IMO it wouldn't be too bad to just add a general purpose `StripeVersion` here to allow per-request overrides even if the library pegs itself to a single version normally.

Reply:
Yeah, I'm not a fan of my solution either, I tried to have the naming be specific enough. Not sure how to offer an extra option because that's the way IdempotencyKey and other features were built. Do you have an idea?

Reply:
> Do you have an idea?

IMO just renaming it to `StripeVersion` and enabling per-request overrides if a user really wants to do them is the better option.

Reply:
@brandur-stripe I disagree with this approach. Doing this would "pretend" you can set a specific version when really you can't and almost all objects would break right? @grey-stripe handled this in a custom way in Go by catching errors on JSON deserialization specifically for the ephemeralkeys package but I'm not sure how to approach this in stripe-dotnet and whether it's the right approach (I don't think it is).
I guess the other solution is to take the Go approach and have a custom parameter on the options class that is not associated to a JSON key but that seems dirty too.

Reply:
> I disagree with this approach. Doing this would "pretend" you can set a specific version when really you can't and almost all objects would break right?

In reality almost everything would still work, even if you passed an old API version. Not all that much changes version to version, and for the majority of the resources in the API, nothing change at all.

Now that I think about it a little more though, why don't we just leverage visibility here? I think we should still call it `StripeVersion` because having a field specific to ephemeral keys here is pretty ugly, but just mark it `internal` so that no one but us can use it.

Reply:
Gotcha. I agree that in most cases it would just work, which makes it more dangerous because it's quite hard to catch in tests. But it's now flagged as internal with a clear comment so hopefully no one plays with it incorrectly